File: DCES_API/src/ru/ipo/dces/clientservercommunication/AdjustUserDataRequest.java
package ru.ipo.dces.clientservercommunication;

/**
 * Created by IntelliJ IDEA.
 * User: Илья
 * Date: 11.01.2009
 * Time: 18:34:37
 */
public class AdjustUserDataRequest implements Request {

    public int userID;
    public String sessionID;

    public String login;
    public String password;
    public UserDescription.UserType newType;
    public String[] userData;
}

File: Server/tests/data/AdjustUserDataRequest.php
<?php
class AdjustUserDataRequest{
    public $userID;
    public $sessionID;
    public $login;
    public $password;
    public $newType;
    public $userData;

    function __construct(){
        $this->userID = 0;
        $this->sessionID = null;
        $this->login = 'test_login';
        $this->password = 'test_password';
        $this->newType = 'Participant';
        $this->userData = array();
    }
};
?>
